src/components/organisms/WithNavigationBar/WithNavigationBar.tsx

```diff
@@ -8,22 +8,18 @@ import "./WithNavigationBar.scss";
 interface PropsType {
   children: React.ReactNode;
   redirectionUrl?: string;
-  fullscreen?: boolean;
```

Comment:
It can, and it should be tested, but I don't expect it to happen since this is a legacy leftover. fullscreen was setting the height and width to 100%. Since we have changed that behavior at the root(to actually be 100%), I don't think we need that anymore